src/datanode/src/store.rs

```diff
@@ -138,35 +137,38 @@ pub(crate) fn clean_temp_dir(dir: &str) -> Result<()> {
     Ok(())
 }
 
+/// FIXME: we need to use reqwest 0.12 here.
```

Comment:
Somehow related to the hyper-1.0 upgrade. I'd take a look or postpone it as a follow-up.

Reply:
This could be a massive project; I spent an entire workweek on it.

![image](https://github.com/GreptimeTeam/greptimedb/assets/5351546/07626464-c188-4711-b38c-e54e39173ec6)

Reply:
Somehow make it work at https://github.com/GreptimeTeam/greptimedb/pull/4037/commits/ad0793b48cb0ac19fbd7c52be215140d96ca6eb5.

We still need to bump axum, tonic, hyper, http and more. But bridge reqwest seems separated.